devices/Sim7080/README.md

```diff
@@ -0,0 +1,85 @@
+# SIM7080G - Dual Mode Wireless Module CatM
+
+The 'SIM7080G' supports both 'CAT-M' and 'NB-IoT'. It can be controlled through AT command via a Serial/UART interface.
+
+<mark>The module is tested on a SIM7080G but is also compatible with SIM7070 and SIM7090.</mark>
```

Comment:
we're not adding html elements in the Markdown because those files are then generated automatically. In that vase a suggestion:
```suggestion
> Note: The module is tested on a SIM7080G but is also compatible with SIM7070 and SIM7090.
```

Reply:
Changed!